plugin-viewport/src/main/kotlin/com/mapbox/maps/plugin/viewport/experimental/transition/TransitionUtils.kt

```diff
@@ -0,0 +1,67 @@
+package com.mapbox.maps.plugin.viewport.experimental.transition
```

Comment:
is this file in general in scope for that PR?
many of functions below can be re-used, some comments are outdated. If we're not focused on impl in this PR perhaps we could add them later

Reply:
Will try to remove all the unused utilities